### DISCUSSION TITLE
Stay Code-Connected! Code to Cloud updates

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#30511] [Actions]
GitHub is on a mission to build technology that makes developers happy and productive. As the #1 CI/CD tool used on GitHub repositories, [GitHub Actions](https://docs.github.com/en/actions) is poised to increase the development velocity and increase software reliability for developers. This week I bring you some thrilling new updates, product releases and events from all corners of Code to Cloud. Let’s go! 🚀 

## [Don’t Fly Solo, Come Fly with Copilot](https://github.com/orgs/community/discussions/29932) - AMA on Discussions ⚡ 
![185383814-28aee88f-daa3-4d68-adf1-0b35faccbe9b](https://user-images.githubusercontent.com/95834130/185627373-424225a9-183c-432d-af99-646ed9fcb82f.png)

It’s time for our first AMA on Discussions! On August 24, 2022, we will be hosting an AMA, featuring [@iamcjwilliams](https://github.com/iamcjwilliams), Copilot Product Manager, and [@blackgirlbytes](https://github.com/blackgirlbytes), Developer Advocate to answer all your Copilot questions. 

There is no need to register--just join in on the discussion, and ask your questions. The AMA will stay open for 24 hours giving you plenty of time.

Got questions ❓  on top of your mind? Post them in the comments below and we’ll be sure to answer them.

## GitHub Discussions is now available on GitHub Enterprise Server 🥳 

GitHub is well-known for its community and contributions to the open source world. We are always on the lookout for working on new awesome tools and we’re excited to announce that GitHub Discussions is available on GitHub Enterprise Server. Take a look at [this](https://github.blog/2022-08-17-github-discussions-is-now-available-on-github-enterprise-server/) post by [@mmileski](https://github.com/mmilesk) to know more about what you can do with GitHub Discussions to boost collaboration on your team. 💥 

We’ve also announced general availability of [GitHub Enterprise Server 3.6](https://github.blog/2022-08-17-github-enterprise-server-3-6-is-now-generally-available/) powering Discussions, which comes with a host of improvements for developers, security and administration teams, this update makes developing secure software easier for everyone. If you are an admin or a maintainer of a repository, you can [enable GitHub Discussions ](https://docs.github.com/en/discussions/quickstart)under “Features” in the repository settings. Happy discussion-ing! 😃 

## GitHub Universe 2022 🌠 
![1660238033678](https://user-images.githubusercontent.com/95834130/185627529-4ac7487c-24e0-43b5-aace-9c222e82dca6.png)

Are you ready for GitHub Universe 2022? It's just three short months away! Okay, that might actually be plenty of time to get ready, but we've got tons of great announcements between now and then, so get it on your calendar and [stay tuned](https://www.githubuniverse.com/?utm_medium=org_social&utm_source=linkedin&utm_campaign=LinkedIn-Newsletter-811&scid=&c_1314200=org_social&c_1314201=linkedin&c_1314202=LinkedIn-Newsletter-811) for more!

## Featured: Project ReadMe 🔖 

The ReadME Project amplifies the voices of the developer community by telling stories about: Open Source, Culture, Security, DevOps, and Security. This project is dedicated to highlighting the best from the open source software community.

Hate repetitive tasks while coding? This month we bring you an exemplary blogpost by James Turnbull explaining how using GitHub Actions can help you automate away frustration. See how to [build a CI/CD workflow with Github Actions](https://github.com/readme/guides/sothebys-github-actions) ✍️ 

## ReadMe is now a GitHub secret scanning partner 🤝 

Talking about the Project ReadMe, we’ve also partnered with [ReadMe](https://readme.com/) to scan for their API keys and help secure our mutual users on public and private repositories. `rdme` is ReadMe's official command-line interface (CLI) and [GitHub Action](https://github.com/marketplace/actions/rdme-sync-to-readme#github-actions) wrapper that allows you to effectively perform programmatic updates using the ReadMe API.

 
### Code Security 🔐 

- [Dependabot now alerts for vulnerable GitHub Actions](https://github.blog/2022-08-09-dependabot-now-alerts-for-vulnerable-github-actions/) 🔔 
GitHub will send a Dependabot alert for vulnerable GitHub Actions, making it even easier to stay up to date and fix security vulnerabilities in your Actions workflows.

- [Linking Packages to their source and build using Request to Comment](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/) (npm) 💬 
We’re opening a new [request for comments (RFC)](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/), which discusses linking a package with its source repository and its build environment.

### Copilot 🧑‍✈️ 

- [Initial HTTP proxy support for Copilot](https://github.com/community/community/discussions/29127/)
We’re happy to announce that the latest release of Copilot for Visual Studio Code and Copilot for IntelliJ contains preliminary support for connecting through HTTP proxy servers. The current version supports basic HTTP proxy setups, with and without basic authentication.

### [The Next Step for LGTM.com : Github CodeScanning](https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/) 

Beginning 16th of December 2022, LGTM.com will shut down as we migrate over to [GitHub Code Scanning](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning) . On the 15th of August, we announced the gradual deprecation of LGTM.com on the GitHub blog 📝. For more information, please take a look at the [post](https://github.blog/2022-08-15-the-next-step-for-lgtm-com-github-code-scanning/) — it contains key dates and information about migrating to GitHub Code Scanning. 🔒 

### [Advisory Database supports GitHub Actions advisories](https://github.blog/changelog/2022-08-09-advisory-database-supports-github-actions-advisories/) 👨‍💻 
The [GitHub Advisory Database](https://github.com/advisories/) now includes curated security advisories for vulnerabilities on [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). This means, If you have a dependency on any vulnerable GitHub Actions, GitHub will send Dependabot alerts over the coming days, making you less vulnerable to malicious code.

### Actions and Packages 💯 

- [GitHub Actions: Self-hosted runners now support Apple M1 hardware](https://github.blog/changelog/2022-08-09-github-actions-self-hosted-runners-now-support-apple-m1-hardware/) 🤩 
Actions runner support for Apple silicon hardware, such as the M1 chip, is now generally available. This provides teams with the capability to run self-hosted macOS workflows in a macOS ARM64 runtime. For additional information on how to set up a self-hosted macOS ARM64 runner, please refer to our [documentation](https://docs.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-default-labels-to-route-jobs). 🏃 
Further to this change, the repository that houses the images installed on [GitHub-hosted runners has been renamed](https://github.blog/changelog/2022-08-09-github-actions-self-hosted-runners-now-support-apple-m1-hardware/) from `actions/virtual-environments` to `actions/runner-images`. These images are maintained by GitHub and used by GitHub Actions.

- [GitHub Actions: Ubuntu 22.04 is now generally available on GitHub-hosted runners](https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/) 🆕 
Ubuntu 22.04 is now generally available on GitHub-hosted runners. To use it now, simply add `runs-on: ubuntu-22.04` or `runs on: ubuntu-latest` in your workflow file

## Actions Pro-tip ⚓ 
If you’re someone who extensively uses GitHub Projects for personal productivity. Here’s a recommendation: Use [this GitHub Action](https://github.com/actions/add-to-project) to automatically add newly created issues/pull requests to your projects. That way you’re always up to date.

Remember you can always find heaps of Actions, over on the [GitHub Marketplace](https://github.com/marketplace). 🛒 

### [5 simple things you can do with GitHub Packages to level up your workflows](https://github.blog/2022-08-04-5-simple-things-you-can-do-with-github-packages-to-level-up-your-workflows) 🏅 

From hosting private packages in a private repository to tightening your security profile with GITHUB_TOKEN, checkout [@tinaheidinger](https://github.com/tinaheidinger) explaining five simple ways by which you can streamline your workflow with [GitHub Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages). 

Want to keep track of what’s coming? Subscribe to all Changelog posts via [RSS](https://github.blog/changelog/feed/) or follow GitHub Changelog on [Twitter](https://twitter.com/ghchangelog) to stay updated on everything we ship. 🚢 

If you ever need any help, [create a new Discussion thread](https://github.com/orgs/community/discussions/new) in the communities or contact [GitHub Support](http://support.github.com/) which is [now available in 7 new languages](https://github.blog/changelog/2022-08-17-github-support-portal-is-now-available-in-7-new-languages/). ☮️

**Comment by SE-LAPS** [comment]
https://youtu.be/3x-ev353jfA

**Comment by Vana12330** [comment]
http://telegram.me/Vas1bot
